src/Algebra/Structures/Pointed.agda

```diff
@@ -0,0 +1,49 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Defines `IsPointedMonoid`
+-- intermediate between `Monoid` and `SemiringWithoutAnnihilatingZero`
+--
+-- By contrast with the rest of `Algebra.Structures`, this is modelled
+-- on an underlying `RawMonoid`, rather than a 'flattened' such signature
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+open import Algebra.Bundles.Raw
+import Algebra.Structures as Structures
+open import Data.Nat.Base as ℕ using (ℕ)
+open import Data.Unit.Base
+open import Level using (Level; _⊔_; Lift)
+
+import Algebra.Definitions.RawMonoid as Definitions
+
+module Algebra.Structures.Pointed where
+
+private
+  variable
+    c ℓ : Level
+
+
+------------------------------------------------------------------------
+-- Structures with 1 binary operation & 2 elements
+------------------------------------------------------------------------
+
+record IsPointedMonoid
+  (rawMonoid : RawMonoid c ℓ)
+  (• : RawMonoid.Carrier rawMonoid) : Set (c ⊔ ℓ)
+  where
+  open RawMonoid rawMonoid public
+  open Structures _≈_
+
+  field
+    isMonoid : IsMonoid _∙_ ε
+
+  _×• : ℕ → Carrier
+  n ×• = n × • where open Definitions rawMonoid
+
+  module Literals where
```

Comment:
Why define this as a named module, and not just a `Number` record directly?

Reply:
* I had stymied myself with imports from `Agda.Builtin.FromNat`, perhaps in the mistaken belief that that was how `Number` instances get introduced...
* I had (for some reason) wanted to distinguish between the declaration of a module defining the implementation, from its export via a `number : Number ...` definition elsewhere...